build.xml

```diff
@@ -27,7 +27,7 @@
     <property name="debuglevel" value="source,lines,vars"/>
 
     <!-- default version and SCM information -->
-    <property name="base.version" value="4.0-rc1"/>
+    <property name="base.version" value="4.0.700-alpha3"/>
```

Comment:
Will it generate conflict every time we will pull?
Or is it something to remember / put in the pulling doc?